reference/scalability_guide/more_than_1Go_of_product_medias_to_export.rst

```diff
@@ -0,0 +1,39 @@
+More than 1Go of product medias to export?
```

Comment:
Also, although [wikipedia](https://en.wikipedia.org/wiki/Octet_(computing)) actually knows of the term `1Go` in English, I think I've only seen French people actually use it. It might be less confusing for the rest of the world to use `1GB` or something like that?